src/main/java/training/endava/skeleton/MainApplication.java

```diff
@@ -1,64 +1,223 @@
+//        System.out.println(pr.findAll());
+//
+//        System.out.println("-----------------------------------------------------------------------------------------");
+////        pr.deleteById(9);
+////        System.out.println(persons);
+//
+//        System.out.println("-----------------------------------------------------------------------------------------");
+//        System.out.println("Count: " + pr.count());
+//
+//        System.out.println("-----------------------------------------------------------------------------------------");
+//        pr.deleteAll();
+//        System.out.println(persons);
 
-        PersonRepository pr=new PersonRepository();
 
-        System.out.println("-----------------------------------------------------------------------------------------");
-        pr.save(p8);
-        System.out.println(persons);
+        //###################### Company DAY 5#########################.
 
-        System.out.println("-----------------------------------------------------------------------------------------");
-        System.out.println(pr.findById(8));
+        List<Person> persons1 = new ArrayList<>();
+        Person p11 = new Person(1, "Gigel1", "07XXXXXXX1", "Iasi");
+        Person p21 = new Person(2, "Gigel2", "07XXXXXXX2", "Botosani");
+        Person p31 = new Person(3, "Gigel3", "07XXXXXXX3", "Suceava");
 
-        System.out.println("-----------------------------------------------------------------------------------------");
-        System.out.println("Person with id 8 exist in db: " + pr.existsById(8));
+        persons1.addAll(Arrays.asList(p11, p21, p31));
 
-        System.out.println("-----------------------------------------------------------------------------------------");
-        System.out.println(pr.findAll());
+        List<Person> persons2 = new ArrayList<>();
+        Person p111 = new Person(1, "Gigel1", "07XXXXXXX1", "Iasi");
+        Person p211 = new Person(2, "Gigel2", "07XXXXXXX1", "Botosani");
+        Person p311 = new Person(3, "Gigel3", "07XXXXXXX1", "Suceava");
 
-        System.out.println("-----------------------------------------------------------------------------------------");
-//        pr.deleteById(9);
-//        System.out.println(persons);
+        persons1.addAll(Arrays.asList(p111, p211, p311));
+
+
+        List<Company> companiesList = new ArrayList<>();
+        Company c1 = new Company("Company 1", 2010, persons, CompanyArea.Finance, 1000000);
+        Company c2 = new Company("Company 2", 2011, persons1, CompanyArea.CustomerSupport, 200000);
+        Company c3 = new Company("Company 3", 2012, persons2, CompanyArea.It, 500000);
+
+        companiesList.addAll(Arrays.asList(c1, c2, c3));
+
+
+
+
+        System.out.println("****************************************************************************");
+        //1. List the names of all companies.
+
+        List<String> namesOfAllCompanies = companiesList.stream()
+                .map(p -> p.getName())
```

Comment:
It is preferred to use method reference instead. Replace lambdas everywhere is applicable.